Sources/TuistKit/Services/Cloud/CloudProjectListService.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import TSCBasic
+import TuistCloud
+import TuistLoader
+import TuistSupport
+
+protocol CloudProjectListServicing {
+    func run(
+        serverURL: String?
+    ) async throws
+}
+
+final class CloudProjectListService: CloudProjectListServicing {
+    private let listProjectsService: ListProjectsServicing
+    private let cloudURLService: CloudURLServicing
+
+    init(
+        listProjectsService: ListProjectsServicing = ListProjectsService(),
+        cloudURLService: CloudURLServicing = CloudURLService()
+    ) {
+        self.listProjectsService = listProjectsService
+        self.cloudURLService = cloudURLService
+    }
+
+    func run(
+        serverURL: String?
+    ) async throws {
+        let cloudURL = try cloudURLService.url(serverURL: serverURL)
+
+        let projects = try await listProjectsService.listProjects(
+            serverURL: cloudURL
+        )
+        
+        if projects.isEmpty {
+            logger.info("You currently have no cloud projects. Create one by running `tuist cloud project create`.")
```

Comment:
Do we format backslash? What about capitalising "cloud" to make sure we are talking about an area of Tuist? Should we say "Tuist Cloud" or just "Cloud"?

Reply:
Since this is in the context of you running `tuist cloud project list`, I'd think `Cloud` might be enough. I can capitalize it, though 👍